### PR TITLE
Init web app prop

### PR DESCRIPTION
**Pull request by jdneo**
Initialize the web app property view page.

In Eclipse, add one [line](https://github.com/jdneo/azure-tools-for-java/blob/2ff0af576a60e98d8d525c68d2ba45edcf6d3a08/PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/views/ServiceExplorerView.java#L331) to turn off this feature, and should be removed after this feature is finished.

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/views/ServiceExplorerView.java [hunk 1]
```diff
@@ -322,8 +327,13 @@ public void menuAboutToShow(IMenuManager manager) {
                     Node node = ((TreeNode) selection.getFirstElement()).node;
                     if (node.hasNodeActions()) {
                         for (final NodeAction nodeAction : node.getNodeActions()) {
-                        	ImageDescriptor imageDescriptor = nodeAction.getIconPath() != null ? Activator.getImageDescriptor("icons/" + nodeAction.getIconPath()) : null;
+                            // TODO: remove this after web app property in Eclipse is finished
+                            if (node instanceof WebAppNode && "Show properties".equals(nodeAction.getName())) {
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/views/ServiceExplorerView.java]: move hard code to const

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java [hunk 2]
```diff
@@ -370,6 +373,29 @@ public void openContainerRegistryPropertyView(@NotNull ContainerRegistryNode nod
         }
     }
 
+    @Override
+    public void openWebAppPropertyView(@NotNull WebAppNode webAppNode) {
+        String webAppName = webAppNode.getName();
+        String sid = webAppNode.getSubscriptionId();
+        String resId = webAppNode.getWebAppId();
+        if (Utils.isEmptyString(sid) || Utils.isEmptyString(resId)) {
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java]: show message here?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java [hunk 3]
```diff
@@ -370,6 +373,29 @@ public void openContainerRegistryPropertyView(@NotNull ContainerRegistryNode nod
...
+            return;
+        }
+        Project project = (Project) webAppNode.getProject();
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java]: need check null before cast?

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java [hunk 4]
```diff
@@ -370,6 +373,29 @@ public void openContainerRegistryPropertyView(@NotNull ContainerRegistryNode nod
...
+        FileEditorManager fileEditorManager = FileEditorManager.getInstance(project);
+        if (fileEditorManager == null) {
+            return;
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/helpers/UIHelperImpl.java]: message?

**Comment by jdneo** [comment]
retest this please

**Pull request merged**